docs/app/Examples/modules/Popup/Triggers/PopupClickExample.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Button, Icon, Popup } from 'stardust'
+
+export default function () {
```

Comment:
In light of #564, let's name example classes/functions `PopupExampleClick`.  Both the filename and default export should have the same name as well.